src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIOTalonFX.java

```diff
@@ -68,41 +80,49 @@ public ShooterIOTalonFX() {
 
     TalonFXConfiguration acceleratorConfig = rightFlywheelConfig;
     acceleratorConfig.CurrentLimits.SupplyCurrentLimit = 40.0;
-    acceleratorConfig.Feedback.SensorToMechanismRatio = ShooterConstants.ACCELERATOR_GEAR_RATIO;
+    acceleratorConfig.Feedback.SensorToMechanismRatio = ShooterConstants.ACCELERATOR_GEAR_REDUCTION;
     accelerator.getConfigurator().apply(acceleratorConfig);
 
     leftPosition = leftFlywheel.getPosition();
-    leftVelocityRadiansPerSecond = leftFlywheel.getVelocity();
+    leftVelocityRotationsPerSecond = leftFlywheel.getVelocity();
+    leftVelocitySetpointRadiansPerSecond = leftFlywheel.getClosedLoopReference();
+    leftVelocityErrorRadiansPerSecond = leftFlywheel.getClosedLoopError();
     leftAppliedVolts = leftFlywheel.getMotorVoltage();
     leftCurrentAmps = leftFlywheel.getSupplyCurrent();
     leftTemperatureCelsius = leftFlywheel.getDeviceTemp();
 
     rightPosition = rightFlywheel.getPosition();
-    rightVelocityRadiansPerSecond = rightFlywheel.getVelocity();
+    rightVelocityRotationsPerSecond = rightFlywheel.getVelocity();
+    rightVelocitySetpointRadiansPerSecond = rightFlywheel.getClosedLoopReference();
+    rightVelocityErrorRadiansPerSecond = rightFlywheel.getClosedLoopError();
     rightAppliedVolts = rightFlywheel.getMotorVoltage();
     rightCurrentAmps = rightFlywheel.getSupplyCurrent();
     rightTemperatureCelsius = rightFlywheel.getDeviceTemp();
```

Comment:
Make sure to instantiate the status signals in the same order that the inputs are created in ShooterIO.java